ocrd_page_to_alto/cli.py

```diff
@@ -7,6 +7,8 @@
 
 @click.command(context_settings=CONTEXT_SETTINGS)
 @ocrd_loglevel
+@click.option('--alto-version', default='4.2', help='Choose version of ALTO-XML schema to produce (older versions may not preserve all features)',
```

Comment:
Wouldn't it be more reliable to determine the ALTO version from the input ALTO XML? And then change `NAMESPACES['alto']` accordingly?